### DISCUSSION TITLE
New plotting BUG

### DISCUSSION THREAD

**Discussion by PatMateluna** [Chia-Network/chia-blockchain#6449] [Support]
I'm plotting with multiple SSD drives, 25 plots at a time, usually when one drive is done, I reformat the drive and start a new Parallel plotting with that drive. Now when I try to start the new plotting I go thru the hole process as usual, and when I create the plot it does not show on the (local harvest plots) where all my other plots are still running. I check the hard drive it has nothing on it, this just started happening few days ago. If I start all 25 plots from the start there is no problem, my CPU runs @86% Memory @62% MAX

Please advice!!

Windows 10 Pro
AMD Ryzen Threadripper 3960X 3.8 GHz 24-Core TRX4 Processor
ASUS Republic of Gamers Strix TRX40-XE Gaming TRX4 ATX Motherboard
ASUS Ryujin 360 Liquid CPU Cooler
CORSAIR vengeance RGB PRO 126GB 4x32 DDR4 3200 (PCA 25600)
6 SSD drives

**Comment by BrandtH22** [comment]
@PatMateluna , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
